aiogram/types/update.py

```diff
@@ -53,3 +53,62 @@ class Update(TelegramObject):
     """*Optional*. The bot's chat member status was updated in a chat. For private chats, this update is received only when the bot is blocked or unblocked by the user."""
     chat_member: Optional[ChatMemberUpdated] = None
     """*Optional*. A chat member's status was updated in a chat. The bot must be an administrator in the chat and must explicitly specify 'chat_member' in the list of *allowed_updates* to receive these updates."""
+
+    @property
+    def event(self) -> Tuple[str, TelegramObject]:
```

Comment:
Can't agree this function naming.

If you call it `property` - it should return property only, not tuple

Reply:
@Olegt0rr, what's wrong with that?